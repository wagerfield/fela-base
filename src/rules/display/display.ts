export interface DisplayRuleOptions {
  display?: any
}

export default ({ display }: DisplayRuleOptions = {}) => ({
  display
})
