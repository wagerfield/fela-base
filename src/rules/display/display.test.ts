import display from "./display"

describe("display(props)", () => {
  it("renders expected snapshots", () => {
    expect(
      display({
        display: "inline"
      })
    ).toMatchSnapshot()
    expect(
      display({
        display: "block"
      })
    ).toMatchSnapshot()
  })
})
