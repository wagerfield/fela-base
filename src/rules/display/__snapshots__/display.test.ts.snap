// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`display renders expected snapshots 1`] = `
Object {
  "display": "inline",
}
`;

exports[`display renders expected snapshots 2`] = `
Object {
  "display": "block",
}
`;
